android/lib/build.gradle

```diff
@@ -16,7 +16,7 @@ android {
 }
 
 dependencies {
-  compile 'com.facebook.react:react-native:0.29.0'
+  compile 'com.facebook.react:react-native:+'
```

Comment:
I figured this would be a no go.  Is there a way we can make this library survive React Native upgrades without it though?